scripts/generate-google3-docs/resource-reference/_toc.yaml

```diff
@@ -291,6 +291,8 @@ toc:
     path: /config-connector/docs/reference/resource-docs/compute/computenetworkfirewallpolicy.md
   - title: "ComputeNetworkFirewallPolicyAssociation"
     path: /config-connector/docs/reference/resource-docs/compute/computenetworkfirewallpolicyassociation.md
+  - title: "ComputeNetworkFirewallPolicyRule"
```

Comment:
Remove?

As we don't add a reference doc for ComputeNetworkFirewallPolicyRule, the _toc file change should not be done, otherwise it might cause confusion to public doc readers.

Though if you believe the reference doc is important, then we can work together to support the doc.

Reply:
Removed